src/main/java/bio/terra/configuration/ApplicationConfiguration.java

```diff
@@ -67,4 +67,12 @@ public void setUserEmail(String userEmail) {
         this.userEmail = userEmail;
     }
 
+    @Value("${datarepo.dnsname")
```

Comment:
I'd like to avoid `@Value` annotations if we can -- can `ConfigurationProperties` work here?

Reply:
It was a one off and I didn't want to make another module just for that.
One thought - we could make one file that held all of the config properties that we use for our project deployment.

Reply:
Instead of making another module I was thinking we could do the prefix trick like we did with the `oauth` variables in the `SwaggerDocumentationConfig`. Now that I think about it more, this would be weird to mix in with the other uses of `@Value` in this file. 

For env-specific deployment type properties we have been injecting them via environment variables. I think this should go into the api config json that we store in vault for each environment and then we can map it to the right environment variable (`DATAREPO_DNSNAME`) on deploy.